src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -243,19 +622,52 @@
 		"sourcePath" : "log.sagebase.org/fileUploadRecords/",
 		"tableDescriptor": {
 			"name": "fileuploadrecords",
-			"columns": {
-				"timestamp": "timestamp",
-				"user_id": "bigint",
-				"project_id": "bigint",
-				"file_handle_id": "bigint",
-				"association_object_id": "bigint",
-				"association_object_type": "string",
-				"stack": "string",
-				"instance": "string"
-			},
-			"partitionKeys": {
-				"record_date": "date"
+			"description" : "The table contains file upload records. Records contains information regarding upload of a file on Synapse.",
+			"columns":[{
+				"name": "timestamp",
+				"type" : "timestamp",
+				"comment" : "The timestamp of the file upload."
```

Comment:
A file upload is a long running process with multiple steps, retries, etc.  We need to explain what timestamp in the long running process is recorded here.